hbase-server/src/main/jamon/org/apache/hadoop/hbase/tmpl/master/MasterStatusTmpl.jamon

```diff
@@ -673,6 +675,8 @@ AssignmentManager assignmentManager = master.getAssignmentManager();
         <td><a href="table.jsp?name=<% snapshotTable.getNameAsString() %>"><% snapshotTable.getNameAsString() %></a>
         </td>
         <td><% new Date(snapshotDesc.getCreationTime()) %></td>
+        <td><% snapshotDesc.getOwner() %></td>
+        <td><% snapshotDesc.getTtl() %></td>
```

Comment:
Does it make sense to also check if it is 0 and display `FOREVER` as done in `snapshotsStats.jsp`?

Reply:
human readable TTL? or use <th>TTL(Seconds)</th>?

Reply:
I think human readable would be better.